### ISSUE TITLE
Expose the usefull experimental stats to a JavaScript layer

### ISSUE THREAD

**Issue by Doreapp** [w3c/webrtc-stats#609]
Some statistics, such as `freezeCount`, `totalFreezeDuration`, `pauseCount` and `totalPauseDuration` are only available from a native level or through *webrtc-internals*. 

Those are documented as [webrtc-provisional-stats](https://github.com/w3c/webrtc-provisional-stats) and seem to be implemented some time ago (according to [this commit](https://webrtc.googlesource.com/src.git/+/0237106559e63aea6d1ef4b11b64a32413564cdd) in February 2019)

I would love to be able to access these metrics from a JavaScript level, ideally from the `getStats` function.

**Comment by vr000m** [comment]
The experimental stats should be available via getStats. In the actual browser implementation, it might be available via origin-trials. @henbos can throw some light on the status of the Freeze* metrics in Chrome.

**Comment by henbos** [comment]
We have exposed experimental metrics in the past in origin trials but I don't think these metrics are tied to any ongoing experiments afaik.

Most likely these metrics were added by native users due to "probably being useful" but not having any strong justification or backing by the spec. So either we never followed up on them after them having been proven useful or alternaively maybe they weren't useful.

But probably we just forgot about them :)
